packages/tooltip/src/main.js

```diff
@@ -191,6 +191,10 @@ export default {
         clearTimeout(this.timeoutPending);
       }
       this.showPopper = false;
+
+      if (this.disabled) {
```

Comment:
```js
if (this.disabled && this.popperOptions.removeOnDestroy) {
```

Reply:
is it necessary?

Reply:
Yes, we should only remove DOM when `this.popperOptions.removeOnDestroy` is true. Otherwise, the user may have some DOM manipulation even when the Tooltip is hidden, and `doDestroy` here may lead to some unexpected errors.

Reply:
> we should only remove DOM when this.popperOptions.removeOnDestroy

It is implemented inside "popper.js"  https://github.com/ElemeFE/element/blob/dev/src/utils/popper.js#L213

Reply:
Ah, you're right.